crates/store/src/genesis/mod.rs

```diff
@@ -26,6 +26,7 @@ pub mod config;
 /// Represents the state at genesis, which will be used to derive the genesis block.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct GenesisState {
+    pub fee_parameters: FeeParameters,
     pub accounts: Vec<Account>,
```

Comment:
nit: I would reverse the order of fields - first `accounts` and then `fee_parameters`.